lib/apiservers/engine/backends/volume.go

```diff
@@ -67,14 +69,21 @@ func NewVolumeModel(volume *models.VolumeResponse, labels map[string]string) *ty
 type Volume struct {
 }
 
+// acceptedVolumeFilters are filters that are supported by VIC
+var acceptedVolumeFilters = map[string]bool{
+	"dangling": true,
+	"name":     true,
+	"driver":   true,
+}
+
 // Volumes docker personality implementation for VIC
 func (v *Volume) Volumes(filter string) ([]*types.Volume, []string, error) {
 	defer trace.End(trace.Begin("Volume.Volumes"))
 	var volumes []*types.Volume
 
 	client := PortLayerClient()
 	if client == nil {
-		return nil, nil, derr.NewErrorWithStatusCode(fmt.Errorf("Failed to get a portlayer client"), http.StatusInternalServerError)
+		return nil, nil, derr.NewErrorWithStatusCode(fmt.Errorf("failed to get a portlayer client"), http.StatusInternalServerError)
```

Comment:
This err string is mentioned 4 times in this file.  How about we make this a const or a var and return that instead?